tests/extras/primitives/primitives.test.js

```diff
@@ -340,31 +340,6 @@ suite('registerPrimitive (using innerHTML)', function () {
     });
   });
 
-  test('handles primitive created and updated in init method', function (done) {
```

Comment:
This test was specifically introduced to cover a regression, however I now believe the covered behaviour is actually a bug. The partial combining of object-based single-property data only occurs when applied _before_ the entity is initialized, not after. This inconsistent behaviour isn't ideal, so either it should always combine or never.

If backwards compatibility to this extend is needed, I can introduce a quirk that explicitly causes the old behaviour when updating a object-based single property before initialization.